src/einsteinpy/coordinates/utils.py

```diff
@@ -6,6 +6,16 @@
 _c = constant.c.value
 
 
+class Error(Exception):
```

Comment:
Having __str__ return only the class/error name might be problematic. The raised Error on the terminal will then look like : `einsteinpy.coordinates.utils.CoordinateError:  CoordinateError` .
Currently it looks something like this :
`einsteinpy.coordinates.utils.CoordinateError: Metric object has been instantiated with a coordinate system, ( Spherical ) other than Cartesian Coordinates.`

Reply:
![image](https://user-images.githubusercontent.com/32370609/104089892-926c2680-5298-11eb-892f-c3a8693085a5.png)

This is what i'm doing. Is this okay?

Reply:
@Aditya7799 As you may know, the purpose of `str` is to be readable, while that of `repr` is to be unambiguous. One way to be "readable" is to reduce clutter. Many of the details, output using `repr` can clutter `stdout` or `logfile`. Hence my suggestion. However, with EinsteinPy, we have mostly kept the outputs same for both `str` and `repr`. So, let's continue with this "tradition" and do something like:
```
def __repr___(self):
    return f"{self.__class__.__name__} : [self.message}"

def __str__(self):
    return self.__repr__()
```
Also, a small suggestion to use f-strings everywhere, since EPy has already dropped support for `python<=3.7`. f-strings are faster than `.format()` or `+` or `%s` substitution.

Reply:
Please see this: https://stackoverflow.com/a/60465422/11922029. Serializability will be important (hopefully) pretty soon. So, `__str__` and `__repr__` will be needed as well. It's better to model the `BaseError` class on that SO answer.

Reply:
Another thing, let's call the base Error class - `BaseError` instead of `Error`.